src/components/Card/CardItem.tsx

```diff
@@ -38,9 +38,14 @@ export function CardItem({
           <Card.Title>{title}</Card.Title>
           <Card.Text>{text}</Card.Text>
           <Card.Footer className="mt-auto">
-            <a href={link} target="_blank" rel="noreferrer">
+            <Card.Link
+              href={link}
+              target="_blank"
+              rel="noreferrer"
```

Comment:
Does it warn that we need `rel="noreferrer"`? It wasn't showing me that warning before and I figured there's some magic happening in the `Card.Link` component.

Reply:
Oh, I kept it because it was included in yours, it seemed like a sound choice, and when I did a search to see if that setting comes as a part of `Card.Link`, I did not find a quick answer.

But no, when I remove it, I do not get any warnings.

Reply:
The internet is a weird and wonderful place.